posthog/temporal/batch_exports/batch_exports.py

```diff
@@ -241,6 +255,9 @@ async def iter_records_from_model_view(
     if filters_str:
         filters_str = f"AND {filters_str}"
 
+    # TODO: this can be simplified once all backfill inputs are migrated
+    is_backfill = (backfill_details is not None) or is_backfill
```

Comment:
In case we refactor this before removing it, I'd rather call `get_is_backfill` here so that we would only have to refactor one place:
```
```suggestion
    is_backfill = backfill_details.get_is_backfill()
```